packages/truffle/webpack.config.js

```diff
@@ -118,6 +118,11 @@ module.exports = {
           path.resolve(__dirname, "../environment")
         ],
         use: "shebang-loader"
+      },
+      {
+        //needed to get things working with lerna 4.0
```

Comment:
Odd, **Lerna** is currently not at v4.0 on this branch.
```
$(yarn bin)/lerna --version
3.19.0
```

Reply:
Yeah, it's possible other things might cause this too.  IDK.